src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+@media (min-width: 800px) {
+
+    .bg-sustainable-change {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-sustainable-change.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 55% auto;
+    }
+
+    .bg-innovation {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-innovation.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 45% auto;
+    }
+
+    .bg-technology {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-technology.svg');
+        background-position: center bottom 30px;
+        background-repeat: no-repeat;
+        background-size: 70% auto;
+    }
+
+    .bg-business-alignment {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-business-alignment.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-people-culture {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-people-culture.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 60% auto;
+    }
+
+    .bg-risk-management {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-risk-management.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-testimonials {
+        background:url('{{site.baseurl}}/assets/custom/img/softmod/bg-testimonials.jpg');
+        background-size: 110% auto;
+        background-position-y: 360px;
+        background-repeat: no-repeat;
+    }
+}
+
+.bg-sheen {
+    position:relative;
+}
+
+.bg-sheen:before {
+    content:'';
+    background: rgb(252,249,241);
+    background: linear-gradient(34deg, rgba(252,249,241,1) 52%, rgba(252,244,244,1) 100%);
+    position: absolute;
+    width:100%;
+    height: 100%;
+    top: 0;
+    z-index: 0;
+    right: 0;
+}
+
+@media (min-width: 1200px) {
+
+    .bg-sheen:before {
+        right: 100px;
+    };
+
+    .bg-sheen:after {
+        content: '';
+        right: 100px;
+        background-color:#293341;
+        position: absolute;
+        width:100%;
+        height: 450px;
+        bottom: 0;
+        z-index: 0;
+        right: 0;
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-slate-pattern-2.jpg');
+        background-size: 100% auto;
+        background-repeat: no-repeat;
+    };
+}
+
+/********************************
+    Helpers - Positioning
+********************************/
+
+.z-index-1 {
+    z-index: 1;
+    position: relative;
+}
+
+.p-relative {
+    position: relative;
+}
```

Comment:
There are a lot of "helpers" that are clearly available for use across multiple components. This moves the responsibility of styling into the HTML because that's where these "helpers" are applied. 

Let's think about moving towards a methodology where styles and the markup they apply to live closely together.